encord/objects/ontology_labels_impl.py

```diff
@@ -403,6 +403,14 @@ def classification_hash(self, v: Any) -> NoReturn:
     def ontology_item(self) -> Classification:
         return deepcopy(self._ontology_classification)
 
+    @property
+    def classification_name(self) -> str:
+        return self._ontology_classification.attributes[0].name
+
+    @property
+    def object_feature_hash(self) -> str:
```

Comment:
Why are we introducing this new name instead of just using `feature_hash`?

Reply:
Not exactly new, this is how it is called in the log line that triggered the whole chain: 
```
            f"classification_name={self._ontology_classification.attributes[0].name}, "
            f"object_feature_hash={self._ontology_classification.feature_node_hash})"
```
As well as this name being used all over the place in names such as "include_object_feature_hashes". 
But I totally agree it would be better to converge on one naming convention. 
I'm still a bit vague in all of this terminology, so happy to change it other way around and call this `feature_hash` in both property and log line